tests/System/Routes/AddCommentRouteTest.php

```diff
@@ -41,50 +40,73 @@ public function testGivenRequestWithoutParameters_resultIsError() {
 	}
 
 	public function testGivenRequestWithoutTokens_resultIsError() {
-		// TODO: insert donation
-
-		$client = $this->createClient();
-
-		$client->request(
-			'POST',
-			'add-comment',
-			[
-				'kommentar' => 'Your programmers deserve a raise',
-				'public' => '1',
-				'eintrag' => 'Uncle Bob',
-				'sid' => '9001',
-			]
-		);
-
-		$response = $client->getResponse();
-
-		$this->assertTrue( $response->isSuccessful(), 'request is successful' );
-		$this->assertErrorJsonResponse( $response );
+		$this->createEnvironment( [], function( Client $client, FunFunFactory $factory ) {
```

Comment:
@KaiNissen @gbirke pinging you so you both see the new fancy thing (nice way of putting "now not totally fail thing")